### ISSUE TITLE
Compatibility with Boost, Moodle 3.2

### ISSUE THREAD

**Issue by haietza** [gjb2048/moodle-format_grid#45]
Will you be doing an update for Moodle 3.2, the Boost theme? There are a couple of issues currently:
* Grid format modal is cut off on top
![screen shot 2017-02-20 at 1 25 13 pm](https://cloud.githubusercontent.com/assets/669666/23137453/140ed9d6-f770-11e6-8853-d9e375f01785.png)
* Grid format modal gets hidden by nav drawer on left
![screen shot 2017-02-20 at 1 24 43 pm](https://cloud.githubusercontent.com/assets/669666/23137459/1ac81350-f770-11e6-9028-7a7b32d0bf21.png)
* Modals inside grid format modal are greyed out
![screen shot 2017-02-20 at 1 25 25 pm](https://cloud.githubusercontent.com/assets/669666/23137462/1de4f90e-f770-11e6-926b-bce07fe1eab8.png)

**Comment by gjb2048** [comment]
I already have in V3.2.1.0!  What version are you running please?

**Comment by haietza** [comment]
@gjb2048 I apologize, we just updated to 3.2.1.2 (2016111504) and I see that the first two items have been resolved (thank you!). However, the third item (the use of a mod_bootstrapelements modal inside the format_grid modal) is still not working. This is probably not a terribly likely situation, however I came across it in testing the grid plugin and wanted to make you aware of it. It may also be a Bootstrap limitation (modal inside a modal). Thanks for following up!
